### DISCUSSION TITLE
In Search Results webpart to show documents and then list items(V 3.15) and highlighting keyword

### DISCUSSION THREAD

**Discussion by tusharprithviraj** [microsoft-search/pnp-modern-search#2798] [Q&A]
Hi Team,

1. Can we have sequencing or order in search results to show all documents first (docx,pptx,/zip etc ) and then show other results such as list items, pages, external content.

I have tried giving sort by file type but it gets all aspx page first.

Is there a way we can do it.

2. We would also like to highlight the keyword matching in Title, currently it highlights the hithighlighted summary or description.
    When we change the query template query that highlighting the text is completely gone.

Any help Would be much appreciated.

**Comment by kasperbolarsen** [comment]
Perhaps ranking could solve the first issue, see https://www.techmikael.com/2015/01/how-to-boost-metadata-in-sharepoint.html
